pkg/azuredisk/azuredisk_test.go

```diff
@@ -544,6 +544,7 @@ func TestIsAvailabilityZone(t *testing.T) {
 }
 
 func TestIsCorruptedDir(t *testing.T) {
+	skipTestsIfWindows(t)
```

Comment:
do you know how to configure this test in admin mode, actually test unit test could pass in admin mode on Windows:
```
>go test -run TestIsCorruptedDir
--- FAIL: TestIsCorruptedDir (0.00s)
    azuredisk_test.go:555: failed to create curruptedPath: symlink C:\Users\xiazhang\AppData\Local\Temp\csi-mount-test769943179 C:\Users\xiazhang\AppData\Local\Temp\csi-mount-test769943179\curruptedPath: A required privilege is not held by the client.
```

Reply:
could be some setting in windows.yaml config:
https://github.com/kubernetes-sigs/azuredisk-csi-driver/blob/master/.github/workflows/window#L28

Reply:
there is a possibility to use following runas command to solve this issue, while it still prompt password, need to figure it out later:
https://www.tenforums.com/general-support/111929-how-use-runas-without-password-prompt.html

```
C:\Users\xiazhang\go\src\sigs.k8s.io\azuredisk-csi-driver\pkg\azuredisk>runas /user:Administrator "go test -run TestIsCorruptedDir"
Enter the password for Administrator:
```

Reply:
@andyzhangx Just went through this issue thread - https://github.com/golang/dep/issues/773
Seems like its an issue from the `os.Symlink` client library that's preventing the machine from creating Symlinks. 
I tried running the test as administrator from Command promt and it seemed to PASS.

WDYT about skipping this test for now until we figure out a way to run as admin from the workflow?

Reply:
ok